torch_frame/data/multi_tensor.py

```diff
@@ -325,6 +325,37 @@ def select(
     def _single_index_select(self, index: int, dim: int) -> _MultiTensor:
         raise NotImplementedError
 
+    def fillna(self, fill_value: Union[float, int]) -> None:
+        is_float = self.dtype.is_floating_point
+        if (is_float and isinstance(fill_value, int)
+                or not is_float and isinstance(fill_value, float)):
+            raise ValueError(
+                f"Fill value {fill_value}'s dtype does not match {self.dtype}")
+        if is_float:
+            self.values[torch.isnan(self.values)] = fill_value
+        else:
+            self.values[self.values == -1] = fill_value
+
+    def fill_col(
+        self,
+        index: int,
+        fill_value: Union[float, int],
+    ) -> None:
+        """Fill the :obj:`index`-th column in :obj:`MultiTensor` with fill_val.
+
+        Args:
+            index (int): A column index of the tensor to select.
+            fill_value: Any
+        """
+        self._fill_col(index, fill_value)
+
+    def _fill_col(
+        self,
+        index: int,
+        fill_value: Union[float, int],
+    ) -> None:
+        raise NotImplementedError
```

Comment:
Any reason we need `_fill_col` and `fill_col`?

Reply:
Removed.